dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-utilisation-details.test.ts

```diff
@@ -0,0 +1,85 @@
+import { CURRENCY, FeeRecordEntityMockBuilder, FeeRecordUtilisation } from '@ukef/dtfs2-common';
+import { when } from 'jest-when';
+import { getUtilisationDetails } from './get-utilisation-details';
+import { aTfmFacility, aUtilisationReport } from '../../../../../../test-helpers';
+import { TfmFacilitiesRepo } from '../../../../../repositories/tfm-facilities-repo';
+import { NotFoundError } from '../../../../../errors';
+import { mapToFeeRecordUtilisation } from './map-to-fee-record-utilisation';
+
+jest.mock('../../../../../repositories/tfm-facilities-repo');
+jest.mock('../../../../../helpers');
+jest.mock('./map-to-fee-record-utilisation');
+
+describe('get-utilisation-details', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('getUtilisationDetails', () => {
+    describe('when no tfm facility is found for a fee record', () => {
+      it('should throw', async () => {
```

Comment:
should throw a NotFoundError?

Reply:
Not actually in this PR but a very small change so have added